### ISSUE TITLE
2.4.0rc0 g_value_set_boxed: assertion 'G_VALUE_HOLDS_BOXED (value)' failed   initialized, argv = Gdk.init_check(sys.argv)

### ISSUE THREAD

**Issue by vinartis** [PaddlePaddle/Paddle#47206]
### bug描述 Describe the Bug

在2.4.0rc0 版本
https://www.paddlepaddle.org.cn/documentation/docs/zh/2.4rc/guides/beginner/data_load_cn.html
使用如下示例代码
`from matplotlib import pyplot as plt

for data in train_dataset:
    image, label = data
    print('shape of image: ',image.shape)
    plt.title(str(label))
    plt.imshow(image[0])    
    break`

报错信息：
shape of image :  (1, 28, 28)
/usr/lib/python3/dist-packages/gi/overrides/Gdk.py:438: Warning: g_value_set_boxed: assertion 'G_VALUE_HOLDS_BOXED (value)' failed
  initialized, argv = Gdk.init_check(sys.argv)
/home/nscoffee/.local/lib/python3.10/site-packages/matplotlib/backends/_backend_gtk.py:146: DeprecationWarning: Gtk.Window.set_wmclass is deprecated
  self.window.set_wmclass("matplotlib", "Matplotlib")

图片未能正常显示出来

操作系统是 deepin23preview 
➜  paddle-exercise git:(master) ✗ python --version
Python 3.10.5
➜  paddle-exercise git:(master) ✗ python -m pip --version
pip 22.3 from /usr/local/lib/python3.10/dist-packages/pip (python 3.10)
➜  paddle-exercise git:(master) ✗ 

### 其他补充信息 Additional Supplementary Information
示例代码改成下面的图片显示框可以正常弹出了
for data in train_dataset:
    image, label = data
    print("shape of image : ",image.shape)
    pit.title(str(label))
    c = pit.imshow(image[0])
    pit.colorbar(c)
    pit.show()
    break

或

for data in train_dataset:
    image, label = data
    print("shape of image : ",image.shape)
    pit.title(str(label))
    pit.imshow(image[0])
    pit.show()
    break

**Comment by zhangbo9674** [comment]
你好，看报错日志是与matplotlib相关的问题，没有反映出paddle的错误

**Comment by vinartis** [comment]
> 你好，看报错日志是与matplotlib相关的问题，没有反映出paddle的错误

嗯是的，是matplotlib的问题。只是想表达 教程里面的例子在有些系统运行不正常，可以把最后的 pit.show()方法加上就好了

**Comment by zhangbo9674** [comment]
感谢反馈的问题！
